api/spec/src/cloud/main.tsp

```diff
@@ -140,3 +140,37 @@ namespace OpenMeterCloud.Entitlements {
 @route("/api/v1/customers")
 @tag("Customers")
 interface Customers extends OpenMeter.Customers {}
+
+namespace OpenMeterCloud.ProductCatalog {
```

Comment:
I suppose the contents of [this](https://github.com/openmeterio/openmeter/pull/1613/files) would also move under this namespace (eventually).
I'm ok with me sorting out the conflicts

Reply:
Yup, moved it in this change.